includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -45,7 +52,7 @@ public function register() {
 	/**
 	 * Gets required Google OAuth scopes for the module.
 	 *
-	 * @since 1.130.0
+	 * @since n.e.x.t
```

Comment:
This shouldn't have been changed. This methods was already released as part of 1.130.0.

```suggestion
	 * @since 1.130.0
```